reports/vamshi99.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: report
+report_year: '2018'
+permalink: /GSoC/2018/StatusReport/VamshiKrishna
+student: Vamshi Krishna Bommerla
+organisation: coala
+organisation_link : https://coala.io
+project: Implement GitMate automations and plugins for coala
+project_link: https://summerofcode.withgoogle.com/projects/#5344316227059712
+tarball: https://rawgit.com/Vamshi99/GSoCReport/master/project.tar
```

Comment:
can you use tar.gz , and add a `sha256sum: ` key with the output from `sha256sum /your/file` .

c.f. https://github.com/coala/projects/issues/627